arbor/include/arbor/simulation.hpp

```diff
@@ -25,8 +27,13 @@ class simulation_state;
 
 class ARB_ARBOR_API simulation {
 public:
+
     simulation(const recipe& rec, const domain_decomposition& decomp, const context& ctx);
 
+    simulation(const recipe& rec,
```

Comment:
Sneakily swapped the order there, but! it's a much better order so please change all to be consistent.